pkg/promql/functions_test.go

```diff
@@ -67,6 +67,6 @@ func TestFunctionList(t *testing.T) {
 
 	for i := range parser.Functions {
 		_, ok := FunctionCalls[i]
-		testutil.Assert(t, ok, (fmt.Sprintf("function %s exists in parser package, but not in promql package", i)))
+		testutil.Assert(t, ok, fmt.Sprintf("function %s exists in parser package, but not in promql package", i))
```

Comment:
This error exists in upstream and should also be rolled back

Reply:
oh, I thought its just us, so I let it be. Anyways, rolling back that too.

Reply:
Done!